client/talk-plugin-commentbox/CommentForm.js

```diff
@@ -82,11 +82,16 @@ export class CommentForm extends React.Component {
     }
   }
 
+  removeNewLine = (string) => {
+    return string.replace(/\n/g, '');
+  }
+
   render() {
     const {maxCharCount, submitEnabled, cancelButtonClassName, submitButtonClassName, charCountEnable, body, loadingState} = this.props;
 
-    const length = body.length;
-    const isRespectingMaxCount = (length) => charCountEnable && maxCharCount && length > maxCharCount;
+    const cleanedBody = this.removeNewLine(body);
+    const length = cleanedBody.length;
```

Comment:
Even better when we just use `trim` e.g. `const length = body.trim().length`.
That way we also prevent empty spaces, tabs and other whitespace characters.